src/client/NuGet.Client.Staging/StagingClient.cs

```diff
@@ -41,15 +42,14 @@ public StagingClient(Uri stagingServiceUri, ITestOutputHelper logger)
 
             _stagingServiceUri = stagingServiceUri;
             _logger = logger;
-            _loggerAdapter = new XUnitLoggerAdapter(_logger);
 
             _httpClient = new HttpClient();
             _httpClient.Timeout = TimeSpan.FromMinutes(5);
         }
 
-        public async Task<JArray> ListUserStages(string apiKey)
+        public async Task<IList<StageListView>> ListUserStages(string apiKey)

```

Comment:
Is this designed as a public API? It should be probably be IReadOnlyList if possible